pysnmp/__init__.py

```diff
@@ -1,5 +1,5 @@
 # http://www.python.org/dev/peps/pep-0396/
-__version__ = '4.4.12'
+__version__ = "5.0.10"
```

Comment:
Can you update it to `5.0.16`? That would be the released version after merge. We'll make it increase automatically soon.